src/libs/PaginationUtils.ts

```diff
@@ -167,7 +172,13 @@ function getContinuousChain<TResource>(sortedItems: TResource[], pages: Pages, g
 
     const pagesWithIndexes = getPagesWithIndexes(sortedItems, pages, getID);
 
-    let page: PageWithIndex;
+    let page: PageWithIndex = {
```

Comment:
Please test the updates in this file to make sure it works the same way as before